catbuffer/schemas/nem/multisig/multisig.cats

```diff
@@ -22,12 +22,12 @@ struct Cosignature
 	multisig_account_address = inline SizePrefixedAddress
 
 # cosignature attached to an aggregate transaction with prefixed size
-struct SizePrefixedCosignature
+struct SizePrefixedCosignatureV1
```

Comment:
i don't think this should have version postfix since no other base types have one (e.g. VerifiableEntity, Transaction)

Reply:
Cosignature needs suffix, since it's technically transaction, so shouldn't this have suffix as well?

Reply:
we don't version SizePrefixedEntity, VerifiableEntity, Transaction, Block, so not sure why this?

Reply:
I don't mind dropping, but there's slight difference here.
All of the ones you've mentioned, are "transparent", SizePrefixedEntity is actual TYPE (entity that is prefixed with size, and type of that entity can be found later)

this one here is not really a type, it's a hack for NEM...
if for some bizarre reason, we'd also have CosignatureV2, how would the 'new' hack struct vs the 'old' hack struct be named?
SizePrefixedCosignature vs SizePrefixedCosignatureV2 ?